### PR TITLE
Add API changes for UserProfile task notifications

### PR DESCRIPTION
**Pull request by emintham**
Refs #SDE-4523

TODO:
- [x] Set milestone label in pull request (SDE version)
- [x] Updated changelog.md
- [x] Completed documentation for the new endpoint.
- [x] Feature has been merged into SDE develop.

### CODE REVIEW

File: source/includes/changelog.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 ## v4.9
 - Change `related_tasks` to include the title, phase, and url of each related task in the Tasks and specific Task endpoints
+- Change `notify_tasks` of UserProfile to include id, name.
```
Comment (hanifvirani) [inline source/includes/changelog.md]: Change `notify_tasks` attribute on UserProfile maybe?

**Comment by hanifvirani** [comment]
For the list attribute, might be better to use 2 objects in the example to make things clearer.

**Comment by hanifvirani** [comment]
+1

File: source/includes/changelog.md (hunk 1 above)
Comment (emintham) [reply source/includes/changelog.md]: Done

**Pull request merged**